grunt/config/wp_deploy.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
 			deploy_tag: false,
 			// Equals about 10MB.
 			max_buffer: 10000 * 1024,
+			tmp_dir: "<%= paths.svnCheckoutDir %>",
```

Comment:
Is there a specific reason to have `tmp_dir` with `_` and `svnCheckoutDir` having camelcased?

Reply:
yes, the external library uses snake casing for its settings while we use camel casing everywhere else. Changing the external library to use camelcase is not an option.